cluster/manifests/polarsignals/daemonset.yaml

```diff
@@ -35,11 +35,18 @@ spec:
             - --debuginfo-strip
             - --debuginfo-temp-dir=/tmp
             - --debuginfo-upload-cache-duration=5m
+	    - --metadata-external-labels=CLUSTER={{ .Cluster.Alias }};ZONE=$(ZONE_NAME);REGION=$(AWS_REGION)
```

Comment:
```suggestion
            - --metadata-external-labels=CLUSTER={{ .Cluster.Alias }};ZONE=$(ZONE_NAME);REGION=$(AWS_REGION)
```

Reply:
Is it correct that the key is UPPERCASE? I assume it should be lowercase like other labels generally is?